source/direct_debit/index.html.md.erb

```diff
@@ -0,0 +1,107 @@
+---
+title: Direct Debit 
+weight: 75
+---
+
+# GOV.UK Pay Variable Direct Debit
+
+Before you read this documentation, you should be familiar with how the
+[GOV.UK Pay API enables users to make
+payments](/payment_flow_overview/#payment-flow). 
+
+If you have already integrated with the GOV.UK Pay API, you can start using 
+Direct Debit quickly. You should: 
+
+* request a GOV.UK Pay Direct Debit test account [from our 
+support team](/support_contact_and_more_information/#contact-us).
+* add support for the Direct Debit endpoints to your existing GOV.UK Pay integration. 
+* sign in to your GOV.UK Pay Direct Debit account and connect your GOV.UK Pay
+and GoCardless accounts. 
+
+## GOV.UK Pay ‘agreements’
+
+In order to support Direct Debit, the GOV.UK Pay API uses 
+‘agreements’. An agreement represents an individual payer’s consent for future
+payments using a particular funding source.
+
+For Direct Debit transactions, GOV.UK Pay fulfils an agreement by registering
+a Direct Debit instruction (also known as a ‘mandate’) with the payer’s bank.
+
+## Synchronous vs asynchronous payment methods
+
+Card payments are a synchronous payment method. When a user
+completes a card payment using GOV.UK Pay, the platform immediately confirms
+to your service whether the payment has succeeded or failed. If a user
+pays by card, you can immediately release your service to the user.
+
+Direct Debit payments are an asynchronous payment method. When a user
+completes a Direct Debit payment process using GOV.UK Pay, the platform makes
+a request to their bank to set up the Direct Debit mandate and transfer the
+funds. It takes several days to confirm the success of the payment. 
+
+## One-off payments using Direct Debit
+
+To set up a Direct Debit mandate to take a single payment:
+
+* create an API token linked to your Direct Debit service
+* follow the instructions in the [‘Payment Flow’ section](/payment_flow)
+
+You cannot take further payments from users using this method without creating
+a new payment. This requires users to re-enter their payment details.
+
+## Recurring payments using Direct Debit
+
+You can create recurring Direct Debit payments using the POST /v1/agreements
+endpoint. 
+
+### On-demand payment agreement
+
+An on-demand (or “variable”) agreement allows you to collect payments of any
+amount at any time. For example, you may want to collect: 
+
+* money each time a service is provided
+* payments where the amounts vary according to service usage
+* regular payments where you do not know in advance how much each payment will be
+
+You are responsible for initiating on-demand payments using the GOV.UK Pay API. 
```

Comment:
I removed this content:

> You could also use the on-demand functionality to simulate a recurring or scheduled Direct Debit payment, by sending an API call on a set schedule. Users would be emailed each time a payment is collected against their mandate.In future, we may provide functionality for scheduled, recurring Direct Debits. This is where your service provides GOV.UK Pay with a schedule for Direct Debits, and GOV.UK Pay executes that schedule with no further input from your service. Your service would specify the Direct Debit type in the agreement_type field.

Is this essential/should it go back in? Some of it feels like we cover it well enough elsewhere; other parts of it I'm not comfortable with including e.g. "In future we may ..."

Reply:
i think the first two sentences are helpful - the rest can go

Reply:
OK, added back